share/spice/symbolab/symbolab.js

```diff
@@ -0,0 +1,48 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_symbolab = function (api_result) {
+
+        // Validate the response (customize for your Spice)
+        if (!api_result || !api_result.solution) {
+            return Spice.failed('symbolab');
+        }
+
+        DDG.require('mathquill', function (){
+            Spice.add({
+                id: "symbolab",
+                name: "Calculator",
+                data: api_result.solution,
+                meta: {
+                    sourceName: "Symbolab.com",
+                    sourceUrl: api_result.solution.url
+                },
+                normalize: function (data) {
+                    return {
+                        latex: data.originalQuery.replace(/ /g, "\\text{ }"),
```

Comment:
problemLatex is the interpreted problem in Latex format. So it might be a better choice. If the solution or problem latex do not contain ^ _ \ then I think its safe to assume we don't have to render it into latex.